### PR TITLE
Improved hyperbahn local starter script. 

### PR DESCRIPTION
**Pull request by malandrew**
Use window names instead of window indices.
Write ~/.hyperbahn-hosts.json file and a ~/tcurlrc file pointing to that hostlist.

### CODE REVIEW

**Comment by Raynos** [comment]
#toomuchcode

#donotwant

**Comment by Raynos** [comment]
Really tidy. lgtm.

File: hyperbahn-dev.sh [hunk 1]
```diff
@@ -1,14 +1,24 @@
 #!/bin/sh 
 set -e
 
+H0="127.0.0.1:21300"
+H1="127.0.0.1:21301"
+
+if [ ! -d ./node_modules ]; then
```
Comment (Raynos) [inline hyperbahn-dev.sh]: Please `which tcurl` and `npm install tcurl --global` if not tcurl.

Please also update README

**Comment by Raynos** [comment]
Nice. Ship it.

File: README.md [hunk 2]
```diff
@@ -53,13 +53,13 @@ To use a service over Hyperbahn:
...
- - `npm install tcurl --global`
  - `./hyperbahn-dev.sh`
 
 This will spawn a 3 window tmux session with a two-node hyperbahn
-cluster running. The third window runs the health checks of both
-nodes to double check that the hyperbahn cluster is healthy.
+cluster running. It will also run `npm install` if necessary and install
+tcurl globally if possible and fallback to local installation if the npm
+global folder is not writeable. The third window runs the health checks of both nodes using tcurl to double check that the hyperbahn cluster is
+healthy.
```
Comment (prashantv) [inline README.md]: nit: why is this on a separate line, but the above line is too long? We should format to match the above line length

Comment (malandrew) [inline README.md]: nice catch. fixed.

**Pull request merged**